build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/GitAbbreviatedCommitHash.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using IOFile = System.IO.File;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public sealed class GitAbbreviatedCommitHash : Git
```

Comment:
Why introduce this new task instead of using the existing `<GitCommitHash/>`?

Reply:
I thought it would be cleaner and simpler, but that's up to you. Should I add a `CommitHash` output parameter to the pre-existing `GitCommitHash.cs` file and use that instead?

Reply:
Please do, otherwise we'll have two tasks running around which are semantically very similar, which will confuse me when doing new additions ("should we use `<GitCommitHash/>` or `<GitAbbreviatedCommitHash/>`?"), especially when `<GitCommitHash/>` *already* had an `AbbreviatedCommitHash` output property.